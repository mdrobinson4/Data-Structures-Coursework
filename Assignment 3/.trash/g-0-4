25 22 1 ;
18 16 ;
19 13 1 ;
19 19 ;
26 12 ;
22 22 ;
23 25 1 ;
22 27 1 ;
21 19 1 ;
23 23 ;
24 26 1 ;
18 23 ;
24 18 ;
12 13 ;
26 27 1 ;
26 19 1 ;
26 25 1 ;
20 19 1 ;
17 12 1 ;
23 12 1 ;
18 27 1 ;
12 12 ;
27 26 1 ;
18 20 ;
17 22 ;
13 22 ;
22 23 1 ;
23 14 ;
13 22 1 ;
22 24 1 ;
22 19 ;
27 22 ;
