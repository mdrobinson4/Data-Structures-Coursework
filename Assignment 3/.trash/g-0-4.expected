0: 12
1: 13(12)1 17(12)1 23(12)1 26(12)1
2: 19(13)1 22(13)1 14(23)1 18(23)1
3: 16(18)1 20(18)1 24(18)1 27(18)1 21(19)1 25(22)1
weight: 14
